services/showcase/api/services/sandbox.py

```diff
@@ -230,3 +236,20 @@ async def accept_invitation(
         sender_id=sender.id,
         connection_id=uuid.UUID(resp["connection_id"]),
     )
+
+
+async def promote_tenant_to_issuer(
+    sandbox_id: uuid.UUID,
+    tenant_id: uuid.UUID,
+    db: AsyncSession,
+) -> AcceptInvitationResponse:
```

Comment:
Don't think this is the correct response type... is there one?

Reply:
fixed, not sure if there is a better response